src/app/zap-templates/zcl/data-model/chip/door-lock-cluster.xml

```diff
@@ -199,6 +202,51 @@ limitations under the License.
+            <description>AliroReaderVerificationKey</description>
+            <access op="read" privilege="administer" />
+        </attribute>
+        <!-- Conformance feature [ALIRO] - optional -->
+        <attribute side="server" code="129" define="ALIRO_READER_GROUP_IDENTIFIER" type="octet_string" length="16" writable="false" optional="true" isNullable="true" apiMaturity="provisional">
+            <description>AliroReaderGroupIdentifier</description>
+            <access op="read" privilege="administer" />
+        </attribute>
+        <!-- Conformance feature [ALIRO] - optional -->
+        <attribute side="server" code="130" define="ALIRO_READER_GROUP_SUBIDENTIFIER" type="octet_string" length="16" writable="false" optional="true" apiMaturity="provisional">
+            <description>AliroReaderGroupSubIdentifier</description>
+            <access op="read" privilege="administer" />
+        </attribute>
+        <!-- Conformance feature [ALIRO] - optional -->
+        <attribute side="server" code="131" define="ALIRO_EXPEDITED_TRANSACTION_SUPPORTED_PROTOCOL_VERSIONS" type="octet_string" array="true" length="2" writable="false" optional="true" apiMaturity="provisional">
+            <description>AliroExpeditedTransactionSupportedProtocolVersions</description>
+            <access op="read" privilege="administer" />
+        </attribute>
+        <!-- Conformance feature [ALBU] - optional if ALIRO is supported -->
+        <attribute side="server" code="132" define="ALIRO_GROUP_RESOLVING_KEY" type="octet_string" length="16" writable="false" optional="true" isNullable="true" apiMaturity="provisional">
+            <description>AliroGroupResolvingKey</description>
+            <access op="read" privilege="administer" />
+        </attribute>
+        <!-- Conformance feature [ALBU] - optional if ALIRO is supported -->
+        <attribute side="server" code="133" define="ALIRO_SUPPORTED_BLE_UWB_PROTOCOL_VERSIONS" type="octet_string" array="true" length="2" writable="false" optional="true" apiMaturity="provisional">
+            <description>AliroSupportedBLEUWBProtocolVersions</description>
+            <access op="read" privilege="administer" />
+        </attribute>
+        <!-- Conformance feature [ALBU] - optional if ALIRO is supported -->
+        <attribute side="server" code="134" define="ALIRO_BLE_ADVERTISING_VERSION" type="int8u" writable="false" optional="true" apiMaturity="provisional">
+            <description>AliroBLEAdvertisingVersion</description>
+            <access op="read" privilege="administer" />
+        </attribute>
+        <!-- Conformance feature [ALIRO] - optional -->
+        <attribute side="server" code="135" define="NUMBER_OF_ALIRO_CREDENTIAL_ISSUER_KEYS_SUPPORTED" type="int16u" writable="false" optional="true" apiMaturity="provisional">
+            <description>NumberOfAliroCredentialIssuerKeysSupported</description>
+            <access op="read" privilege="view" />
+        </attribute>
```

Comment:
```suggestion
        <attribute side="server" code="135" define="NUMBER_OF_ALIRO_CREDENTIAL_ISSUER_KEYS_SUPPORTED" type="int16u" writable="false" optional="true" apiMaturity="provisional">NumberOfAliroCredentialIssuerKeysSupported</attribute>
```
If the privileges are the defaults, as here, you don't have to do the long-form thing and list them.